asset/scss-about/_queries.scss

```diff
@@ -0,0 +1,240 @@
+@media screen and (min-width: 769px) {
+  .mobile_version {
+    display: none;
+  }
+
+  .header_text {
+    width: 100vw;
+  }
+}
+
+@media screen and (min-width: 769px) {
+  /* header section */
+  nav,
+  .header_text {
+    padding: 0%;
+  }
+
+  .destop-version {
+    display: block;
+    background-color: #3e3e3e;
+    color: #efefef;
+    margin: 0%;
+    padding: 0%;
+    height: 3vh;
+    width: 100vw;
+  }
+
+  .icon_list {
+    list-style: none;
+    display: flex;
+    justify-content: end;
+    position: absolute;
+    left: 60%;
+    padding: 0%;
+    align-content: flex-end;
+  }
+
+  .icon_list li {
+    padding: 0 1.5em;
+    font-size: xx-large;
+    font-weight: lighter;
+  }
+
+  /* second navigation */
+  .navigation {
```

Comment:
- [ ] This second navbar and the first one is causing an overflow on the about page. Please fix that! Also, consider the overflow on the home page.
It's so important that the website appears completely professional.

>A screenshot from your project.
![Screenshot](https://user-images.githubusercontent.com/47625407/124310476-889eee00-db6c-11eb-8236-a3bde82a63d1.png)

_**Please refer to the review summary and the given design link for more information.**_

Reply:
thank you for that insight, I will correct that error right now